src/pe/header.rs

```diff
@@ -381,10 +380,27 @@ impl DosHeader {
             pe_pointer,
         })
     }
+
+    /// Parse the DOS stub.
+    ///
+    /// The DOS stub is a small program that prints the message "This program cannot be run in DOS mode" and exits; and
+    /// is not really read for the PECOFF file format. It's a relic from the MS-DOS era.
+    pub fn parse_dos_stub<'a>(bytes: &'a [u8], pe_pointer: u32) -> error::Result<&'a [u8]> {
```

Comment:
I would expect this fucntion to be `pub fn new` in the impl of `DosStub`?

Reply:
Solved in b1b841cea67fd4fbd7d0ae948dafb6b9d66a346f.
Since its trivial, I am making this change request solved.